app/views/events/_event.html.erb

```diff
@@ -0,0 +1,15 @@
+
+<div class = "row event-body"> 
+      <div class="col col-md-3">
+            
+            Created Bye : <%= event.user.name %>

```

Comment:
i don't know how to get to the event show. i used the browser address bar.

Reply:
By clicking the link in description, we can go to the event page.